### DISCUSSION TITLE
Tooltips on UI elements

### DISCUSSION THREAD

**Discussion by Archytas79** [Yellow-Dog-Man/Resonite-Issues#2109] [New Features]
Something to do a more clear job explaining what a thing does in engine in things like the inspector. Could have it be an an auto-on-install option where hovering over a button for a time will display a textbox facing the player that gives insight into what the thing does.

**Comment by Archytas79** [comment]
It'd be really cool to somehow integrate the wiki into it; auto parse component & variable descriptions to be viewable in engine, but this would be a pie in the sky implementation.

**Comment by LexiBasilisk** [comment]
A generalized request for interface tooltips already exists in issue #94.

Regarding inspectors in particular- we would be unlikely to update the existing interface- we would instead make them into a modular system as described in #563.

If we were to integrate tooltips, we would also probably want to utilize our existing locale system so that tooltip strings could be translated to multiple target languages.

Did you have anything in particular you wished to discuss about tooltips, @Archytas79? Typically we only use discussions in Discord for threaded conversation about an existing issue, e.g. in this case, #94. Otherwise, we usually just expect users to make an [issue](https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/new/choose) instead.

**Comment by Archytas79** [comment]
We were discussing it in discord and I was told to make a request here - I did search for tooltips but didn't find it, my apologies ^^
